Sources/Storage/Disk/DiskStorage.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  DiskStorage.swift
+//  StatsdClient-iOS
+//
+//  Created by Nghia Tran on 10/8/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+final class DiskStorage<Element: Codable>: Storage {
+
+    typealias Key = String
+    typealias Item = Element
+
+    // MARK: - Variable
+    private let handler: PersistentHandler
+    private let queue = DispatchQueue(label: "StatsD_DiskStorage", qos: .default, attributes: .concurrent)
+
+    // Count
+    var count: Int {
+        return queue.syncWithReturnedValue { self.handler.getTotal() }
+    }
+
+    // MARK: - Init
+    init(handler: PersistentHandler) {
+        self.handler = handler
+    }
+
+    // Default
```

Comment:
Remove all unnecessary comments

Reply:
I will remove `// Count` comment.

How about `// MARK:` for each section?

Should I remove it due to the Zalora's Code Convention?